Common/API/ApiAlgorithmStatus.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace QuantConnect.API
+{
+    /// <summary>
+    /// An enum representing the possible status of a live running algorithm on QC
+    /// </summary>
+    public enum ApiAlgorithmStatus
```

Comment:
There are a couple values in AlgorithmStatus that have no equivalent in the Api.  In other words, the Api will reject certain values in AlgorithmStatus.  I thought it was better to create a new enum that only held values the Api will accept.